src/test/java/com/google/sps/UserTest.java

```diff
@@ -39,6 +40,67 @@ public void userNotEquals() {
     User userA = new User("John");
     User userB = new User("Not John");
 
-    Assert.assertFalse(userA.equals(userB));
+    Assert.assertFalse((userA).equals(userB));
```

Comment:
consider using assertNotEquals here which may be easier to read.

Reply:
Done

Reply:
was this updated?